inst/assets/BS4/syntax-highlighting.css

```diff
@@ -6,6 +6,11 @@ pre, code, pre code {
   word-wrap: break-word;
 }
 
+.ref-index-aliases pre, code, pre code {
+  word-wrap: normal;
+  word-break: normal;
```

Comment:
I'm not sure that this is the right fix — I think the problem might be that we're applying `work-break` to `<code>` when we only want to apply it to code. Can you try removing changing the previous selector to `pre, pre code` instead?

(And since you're in here, can you please globally rename `word-wrap` to `overflow-wrap`, since that's now the official name)